apps/docs/src/app/core/component-docs/feed-list-item/examples/fli-avatar/fli-avatar-example.component.html

```diff
@@ -0,0 +1,34 @@
+<fd-feed-list aria-label="Feed List and Item with user avatar example">
+    <fd-feed-list-item
+        authorTitle="John Doe"
+        authorLink="http://example.com/"
+        text="Long text, you can toggle view with more/less button. Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+        maxRows="2"
+        ariaLabel="Feed list item example">
+        <fd-feed-list-avatar>
+            <fd-avatar image="https://avatars.dicebear.com/api/human/fe.svg?m=6&b=%23f5f0f0&mood[]=surprised"size="s"></fd-avatar>
+        </fd-feed-list-avatar>
+        <fd-feed-list-action>
+            <button aria-label="action" fd-button fdType="transparent" [compact]="true" glyph="overflow"></button>
+        </fd-feed-list-action>
+        <fd-feed-list-footer>
+            <small fdItemFooterByline>Aug 20, 2020</small>
```

Comment:
Why use camelcase for this directive name but hyphenated for all your other ones?